chain-evm/src/machine.rs

```diff
@@ -20,6 +20,12 @@ use primitive_types::{H160, H256, U256};
 
 use crate::state::AccountTrie;
 
+/// EVM Configuration.
+pub type Configuration = Config;
```

Comment:
The renaming seems to have no purpose and will make the code less greppable. Leave it at `Config`.

Reply:
Left it as `Config`